src/cmd/usb.js

```diff
@@ -43,7 +43,15 @@ module.exports = class UsbCommand {
 							let info = [device, usbDevice.isInDfuMode];
 
 							if (!usbDevice.isInDfuMode){
-								info.push(usbDevice.getDeviceMode());
+								info.push(
+									usbDevice.getDeviceMode()
+										.catch(error => {
+											if (error.message.includes('timeout')){
```

Comment:
While testing [particle-usb/#29](https://github.com/particle-iot/particle-usb/pull/29), I noticed that sometimes timeout exceptions thrown by the library don't have a message :disappointed: It's better to check if `error` is an instance of particle-usb's `TimeoutError`. I also added a timeout option to `getDeviceMode()` – setting it to something like 2-3 seconds will prevent the list command from taking too long if one of the devices is blocked.

Reply:
> ...sometimes timeout exceptions thrown by the library don't have a message...

:joy::sob: whaa?.. but... how? isn't the timeout error coming from [here](https://github.com/particle-iot/particle-usb/blob/04837184cc80734f240eadb5fb2ef3d2c4f08e8c/src/device-base.js#L232)?

...oh maybe it's coming from [here](https://github.com/particle-iot/particle-usb/blob/master/src/device.js#L68-L88) sometimes..?

why not assign a default `message` [here](https://github.com/particle-iot/particle-usb/blob/04837184cc80734f240eadb5fb2ef3d2c4f08e8c/src/error.js#L49) - e.g. 

```js
this.message = this.message || 'Request timeout';
```

Reply:
Yeah, there will be a message in this specific case, but methods that rely on `RequestSender` internally throw `TimeoutError` without any message set :shrug: 

> why not assign a default message

:+1:

Reply:
✅ in https://github.com/particle-iot/particle-cli/pull/588/commits/368c0584c382862c2f25a88d1407124b7bfaa274